### ISSUE TITLE
After I successfully called StartLogout, GetLocalUserId not null and valid

### ISSUE THREAD

**Issue by BestStream** [PlayEveryWare/eos_plugin_for_unity#633]

**Comment by andrew-hirata-playeveryware** [comment]
Hello @BestStream,
I don't I'm not certain I understand the issue.

**Comment by BestStream** [comment]
@andrew-hirata-playeveryware When I start a game and I need to understand whether a player is logged in or not, so I check EOSManager.Instance.GetLocalUserId(). Before login it returns NULL, and after EpicAccountId value. After I call StartLogout() I expect that EOSManager.Instance.GetLocalUserId() will start returning NULL again, but it don't

**Comment by andrew-hirata-playeveryware** [comment]
Oh I see what you mean. This is probably either a 
a) long standing issue that hasn't broken anything so it hasn't been fixed.
b) the result of a refactor
c) A workaround for some odd behavior in one of our samples.

I'll see if I can't get someone on our team to work on this.

**Comment by paulhazen** [comment]
@BestStream I've opened a PR that I believe resolves this issue. Once it is merged into the `development` branch I will make sure this issue is closed out. Thank you for bringing it to our attention! :)

**Comment by paulhazen** [comment]
Now that PR #640 has been merged into `development` I'm closing this issue.
